chain/network/src/peer_manager/network_state/mod.rs

```diff
@@ -392,6 +448,33 @@ impl NetworkState {
         account_id: &AccountId,
         msg: RoutedMessageBody,
     ) -> bool {
+        if tcp::Tier::T1.is_allowed_routed(&msg) {
+            tracing::debug!(target:"test", "got TIER1 message to send");
+            let accounts_data = self.accounts_data.load();
+            for key in accounts_data.keys_by_id.get(account_id).iter().flat_map(|keys| keys.iter())
+            {
+                let data = match accounts_data.data.get(key) {
+                    Some(data) => data,
+                    None => continue,
+                };
+                let conn = match self.get_tier1_proxy(data) {
+                    Some(conn) => conn,
+                    None => continue,
+                };
+                tracing::debug!(target:"test", "found TIER1 proxy");
+                // TODO(gprusak): in case of PartialEncodedChunk, consider stripping everything
+                // but the header. This will bound the message size
+                conn.send_message(Arc::new(PeerMessage::Routed(self.sign_message(
+                    clock,
+                    RawRoutedMessage {
+                        target: PeerIdOrHash::PeerId(data.peer_id.clone()),
+                        body: msg.clone(),
+                    },
+                ))));
+                break;
```

Comment:
shouldn't we return 'true' here?

or do we want to send the message over tier2 too? (if yes - please comment)